vip-helpers/vip-utils.php

```diff
@@ -73,6 +73,9 @@ function wpcom_vip_noncdn_uri( $path ) {
  * @return string HTML
  */
 function vip_powered_wpcom_img_html( $image ) {
+	//Stores string to allow cache breaking if images need replaced
+	$img_version = "20220317";
```

Comment:
Minor nitpick, can we move this closer to line 89 since that's where it'll be used?

Reply:
👍  - that's a really good suggestion. Updated.